src/app.js

```diff
@@ -42,7 +42,13 @@ const IGNORED_WARNINGS = ['slowlog', 'Setting a timer for a long period of time'
 console.ignoredYellowBox = IGNORED_WARNINGS
 
 // Ignore errors and warnings(used for device testing)
-if (ENV.DISABLE_WARNINGS) LogBox.ignoreAllLogs()
+if (ENV.DISABLE_WARNINGS) {
+  const ignoreWarns = [
+    "Module RCTSplashScreen requires main queue setup since it overrides `constantsToExport` but doesn't implement `requiresMainQueueSetup`. In a future release React Native will default to initializing all native modules on a background thread unless explicitly opted-out of.",
```

Comment:
Can we remove some detail from this first pattern? Maybe just match the first line, like "Module RCTSplashScreen requires main queue setup since it overrides `constantsToExport`", so it's less wordy.

Reply:
Ok